src/index.html

```diff
@@ -18,65 +18,95 @@
     <link rel="icon" type="image/png" href="/img/favicon.png"/>
     <title ng-bind="title + ' - H.S.A. Confluente'">H.S.A. Confluente</title>
 
+    <!-- Script to make the navbar hoverable in desktop format -->
```

Comment:
Not really a fan of having the javascript inside of the html file. But it works ;)